<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="14" passed="12">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="31859" started-at="2016-07-02T07:14:43Z" finished-at="2016-07-02T07:15:15Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="31859" started-at="2016-07-02T07:14:43Z" finished-at="2016-07-02T07:15:15Z">
      <class name="com.griggi.app.MainTest">
        <test-method status="PASS" signature="beforeTest(org.testng.ITestContext)[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="beforeTest" is-config="true" duration-ms="2102" started-at="2016-07-02T12:44:43Z" finished-at="2016-07-02T12:44:45Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[org.testng.TestRunner@25f39c4]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="PASS" signature="loginBad1()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="loginBad1" duration-ms="1495" started-at="2016-07-02T12:44:45Z" description="Logging in. - Too long" finished-at="2016-07-02T12:44:46Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginBad1 -->
        <test-method status="PASS" signature="loginBad2()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="loginBad2" duration-ms="1023" started-at="2016-07-02T12:44:46Z" description="Logging in. - Too short" finished-at="2016-07-02T12:44:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginBad2 -->
        <test-method status="PASS" signature="loginBad3()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="loginBad3" duration-ms="987" started-at="2016-07-02T12:44:47Z" description="Logging in. - Invalid" finished-at="2016-07-02T12:44:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginBad3 -->
        <test-method status="FAIL" signature="loginGood2()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="loginGood2" duration-ms="2382" started-at="2016-07-02T12:44:48Z" description="Verifies that after a bad login, user can enter" finished-at="2016-07-02T12:44:51Z">
          <exception class="org.junit.ComparisonFailure">
            <message>
              <![CDATA[expected:<[The password you entered is not valid.]> but was:<[]>]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.junit.ComparisonFailure: expected:<[The password you entered is not valid.]> but was:<[]>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at com.griggi.app.MainTest.loginGood2(MainTest.java:117)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- org.junit.ComparisonFailure -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginGood2 -->
        <test-method status="PASS" signature="loginGood1()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="loginGood1" duration-ms="2236" started-at="2016-07-02T12:44:51Z" description="Logging in. - GOOD" finished-at="2016-07-02T12:44:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginGood1 -->
        <test-method status="PASS" signature="dashboardUICheck1()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="dashboardUICheck1" duration-ms="161" started-at="2016-07-02T12:44:53Z" description="Check Dashboard UI - 1 (Nav)" finished-at="2016-07-02T12:44:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dashboardUICheck1 -->
        <test-method status="PASS" signature="dashboardUICheck2()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="dashboardUICheck2" duration-ms="71" started-at="2016-07-02T12:44:53Z" description="Check Dashboard UI - 2 (Widgets)" finished-at="2016-07-02T12:44:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dashboardUICheck2 -->
        <test-method status="PASS" signature="dashboardUICheck3()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="dashboardUICheck3" duration-ms="40" started-at="2016-07-02T12:44:53Z" description="Check Dashboard UI - 3 (Header)" finished-at="2016-07-02T12:44:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dashboardUICheck3 -->
        <test-method status="PASS" signature="verifyLatestNodeURL()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="verifyLatestNodeURL" duration-ms="338" started-at="2016-07-02T12:44:53Z" description="Verify the url for the lastest node on dashboard" finished-at="2016-07-02T12:44:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLatestNodeURL -->
        <test-method status="PASS" signature="listOfNodes()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="listOfNodes" duration-ms="292" started-at="2016-07-02T12:44:54Z" description="Verify Nav from database - List of nodes" finished-at="2016-07-02T12:44:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- listOfNodes -->
        <test-method status="FAIL" signature="nodeAdminDropdownArrow()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="nodeAdminDropdownArrow" duration-ms="17" started-at="2016-07-02T12:44:54Z" description="Admin node arrow exists" finished-at="2016-07-02T12:44:54Z">
          <exception class="java.lang.AssertionError">
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError
	at org.junit.Assert.fail(Assert.java:86)
	at org.junit.Assert.assertTrue(Assert.java:41)
	at org.junit.Assert.assertFalse(Assert.java:64)
	at org.junit.Assert.assertFalse(Assert.java:74)
	at com.griggi.app.MainTest.nodeAdminDropdownArrow(MainTest.java:206)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- nodeAdminDropdownArrow -->
        <test-method status="PASS" signature="routerSpeed()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="routerSpeed" duration-ms="4564" started-at="2016-07-02T12:44:54Z" description="Router Speed" finished-at="2016-07-02T12:44:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- routerSpeed -->
        <test-method status="PASS" signature="availableQuota()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="availableQuota" duration-ms="4547" started-at="2016-07-02T12:44:59Z" description="Checks the available quota of the user with the DB" finished-at="2016-07-02T12:45:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- availableQuota -->
        <test-method status="PASS" signature="dataAllocation()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="dataAllocation" duration-ms="11265" started-at="2016-07-02T12:45:03Z" description="Tests the data allocation for a user." finished-at="2016-07-02T12:45:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dataAllocation -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:com.griggi.app.MainTest@1174335c]" name="afterTest" is-config="true" duration-ms="167" started-at="2016-07-02T12:45:15Z" finished-at="2016-07-02T12:45:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
      </class> <!-- com.griggi.app.MainTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
